mne/io/tests/test_apply_function.py

```diff
@@ -31,6 +31,12 @@ def printer(x):
     return x
 
 
+def bad_ch_idx(x, ch_idx):
+    """Pass."""
+    assert x[0] == ch_idx
+    return x
```

Comment:
Also, I would define this function inside `test_apply_function_ch_access` since it's the only place it's used.

```
def test_apply_function_ch_access():
    """Test apply function is able to access channel idx."""
    def bad_ch_idx(x, ch_idx):
        assert x[0] == ch_idx
        return x

    data = ...
    raw = ...
    ...
```